governance-and-automation/automated-workflows/inactive-workspaces-admin.md

```diff
@@ -0,0 +1,79 @@
+---
+description: This article explains how to edit the Inactive Workspaces policy. 
```

Comment:
zašto samo edit postojećeg built-in policya?
built-in policy je applied na sve workspaceve kroz default rule (koji se ne vidi u sučelju), ali ti možeš i kreirati nove policye i applyati ručno ili još bolje kroz rule na željene workspaceve. 
svaki rule koji customer napravi će nadjačati defaultni, tako da tu onda do izražaja zapravo dolazi prednost IW policya nad LM starim featureom